nepalingo-web/.gitignore

```diff
@@ -11,6 +11,8 @@ node_modules
 dist
 dist-ssr
 *.local
+#ignoring the .env
+.env
```

Comment:
let's not gitignore the .env file since I don't want each person to need additional setup steps on their local development machine